discovery-provider/src/queries/get_users.py

```diff
@@ -37,9 +37,10 @@ def get_users_and_ids():
             # Create initial query
             base_query = session.query(User)
             # Don't return the user if they have no wallet or handle (user creation did not finish properly on chain)
-            base_query = base_query.filter(
-                User.is_current == True, User.wallet != None, User.handle_lc != None
-            )
+            if "include_incomplete" not in args or not args["include_incomplete"]:
```

Comment:
could be:

```
if not args.get("include_incomplete")
    # ...
```

also... should the base_query include `User.is_current == True` by default?  It can be bad to omit that filter.

Reply:
I should maybe make a comment inline - this `include_incomplete` flag is only used during signup (can't think of any case where we'd care about `is_current` during signup?). also I'm not sure if `is_current` would be True at this point because this is right before setting their wallet and other metadata, which I think could be consolidated into a single EM txn as a future improvement